update balance (#1028)
vesoft-inc/nebula-docs: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md

    All job management commands can be executed only after selecting a graph space.

## SUBMIT JOB BALANCE IN ZONE

The `SUBMIT JOB BALANCE IN ZONE` statement starts a job to balance the distribution of storage partitions in each zone in the current graph space. It returns the job ID. You can use the `REMOVE` option to specify the Storage services that you want to clear for easy maintenance.

For example:

```ngql
nebula> SUBMIT JOB BALANCE IN ZONE REMOVE 192.168.10.102:9779;
+------------+
| New Job Id |
+------------+
| 26         |
+------------+
```

## SUBMIT JOB BALANCE ACROSS ZONE

> The `SUBMIT JOB BALANCE ACROSS ZONE` statement starts a job to balance the distribution of storage partitions across each zone in the current graph space. It returns the job ID. You can use the `REMOVE` option to specify the zones that you want to clear for easy maintenance.

Reviewer: ```suggestion
The `SUBMIT JOB BALANCE ACROSS ZONE` statement starts a job to balance the distribution of storage partitions across all the zones in the current graph space. It returns the job ID. You can use the `REMOVE` option to specify the zones that you want to clear for easy maintenance.
```

## RECOVER JOB

> The `RECOVER JOB [<job_id>]` statement re-executes the jobs that status is `QUEUE`, `FAILED` or `STOPPED` in the current graph space and returns the number of recovered jobs. If `<job_id>` is not specified, re-execution is attempted starting with the earliest job.

Reviewer: ```suggestion
The `RECOVER JOB [<job_id>]` statement re-executes the jobs that status is `QUEUE`, `FAILED` or `STOPPED` in the current graph space and returns the number of recovered jobs. If `<job_id>` is not specified, re-execution is performed from the earliest job and the number of jobs that have been recovered is returned.
```
